plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
+        "SystemCheckWarnDomHelp": "You should turn on the \"dom\" extension (e.g., install the \"php-dom\" and\/or \"php-xml\" package).",
+        "SystemCheckWarning": "Matomo will work normally, but some features may be missing",
+        "SystemCheckWarnJsonHelp": "You should turn on the \"JSON\" extension (e.g., install the \"php-json\" package).",
         "SystemCheckWarnLibXmlHelp": "You should enable the \"libxml\" extension (e.g., \"install the php-libxml\" package) as it is required by other core PHP extensions.",
-        "SystemCheckWarnOpensslHelp": "You should enable the \"openssl\" extension to allow secure updates.",
+        "SystemCheckWarnOpensslHelp": "You should enable the \"OpenSSL\" extension to allow secure updates.",
         "SystemCheckWarnSimpleXMLHelp": "You should enable the \"SimpleXML\" extension (e.g., install the \"php-simplexml\" and\/or \"php-xml\" package).",
         "SystemCheckWinPdoAndMysqliHelp": "On a Windows server you can add the following lines to your php.ini: %s",
         "SystemCheckWriteDirs": "Directories with write access",
         "SystemCheckWriteDirsHelp": "To fix this error on your GNU\/Linux system, try typing in the following command(s)",
-        "SystemCheckZlibHelp": "You need to configure and rebuild PHP with \"zlib\" support enabled, --with-zlib.",
-        "SystemCheckCronArchiveProcess": "Setup Cron",
-        "FasterReportLoading": "faster report loading",
+        "SystemCheckZlibHelp": "Configure and rebuild PHP with the required \"zlib\" support, --with-zlib.",
+        "SystemCheckCronArchiveProcess": "Set up Cron",
+        "FasterReportLoading": "faster report-loading",
         "SystemCheckCronArchiveProcessCLI": "Managing processes via CLI",
-        "SystemCheckPhpSetting": "To prevent some critical issue, you must set the following in your php.ini file: %s",
+        "SystemCheckPhpSetting": "To prevent a critical error, you must set the following in your php.ini file: %s",
         "SystemCheckUpdateHttps": "Update over HTTPS",
-        "SystemCheckUpdateHttpsNotSupported": "Matomo cannot use HTTPS to update, it will fall back to the insecure HTTP update. Check that CURL or allow_url_fopen is supported and that the openssl PHP extension is installed: https:\/\/matomo.org\/faq\/troubleshooting\/faq_177\/.",
+        "SystemCheckUpdateHttpsNotSupported": "Matomo cannot use HTTPS to update, it will fall back to the insecure HTTP update. Check that CURL or allow_url_fopen is supported and that the OpenSSL PHP extension is installed: https:\/\/matomo.org\/faq\/troubleshooting\/faq_177\/.",
         "PhpBinaryCheck" : "64-bit PHP Binary",
-        "PhpBinaryCheckHelp": "32-bit <p> We recommend upgrading to a 64-bit PHP binary at the latest by January 2026 to prevent bugs in the future.</p>",
+        "PhpBinaryCheckHelp": "32-bit <p>It is recommended to upgrading to a 64-bit PHP binary at the latest by January 2026 to prevent future bugs.</p>",
```

Comment:
Suggest reverting as this has grammatical errors and the original is slightly clearer and more consistent with other messaging.

Reply:
```suggestion
        "PhpBinaryCheckHelp": "32-bit <p>Upgrade to a 64-bit PHP binary by January 2026 to prevent future bugs.</p>",
```